### PR TITLE
Add ceil/floor/log10 builtins

### PR DESCRIPTION
**Pull request by DUKaige**

### CODE REVIEW

**Comment by apavlo** [comment]
@DUKaige  can you resolve conflicts?

File: src/execution/sema/sema_builtin.cpp [hunk 1]
```diff
@@ -1332,6 +1332,10 @@ void Sema::CheckMathTrigCall(ast::CallExpr *call, ast::Builtin builtin) {
     case ast::Builtin::Sinh:
     case ast::Builtin::Tanh:
     case ast::Builtin::ACos:
+    case ast::Builtin::ATan:
```
Comment (gengkev) [inline src/execution/sema/sema_builtin.cpp:1335]: atan added twice?

Comment (xuanxuange) [reply src/execution/sema/sema_builtin.cpp:1335]: resolved

**Comment by jkosh44** [comment]
This PR can now be closed and replaced with ~~https://github.com/cmu-db/terrier/pull/1084~~ https://github.com/cmu-db/terrier/pull/1090

**Pull request closed without merging**